package.json

```diff
@@ -26,7 +26,7 @@
     "postinstall": "husky install",
     "precommit": "lint-staged",
     "test": "yarn workspace @se-2/hardhat test",
-    "vercel": "vercel"
+    "vercel": "yarn workspace @se-2/nextjs vercel"
```

Comment:
Should we add back also this? => `"vercel:yolo": "yarn workspace @se-2/nextjs vercel:yolo"`

Reply:
no, it was removed from nextjs package https://github.com/scaffold-eth/se-2-challenges/pull/39 , so we don't need it in main package.json too. btw works for other challenges too

Reply:
Yeah 🙌, `yarn vercel` is equivalent to `yarn vercel:yolo` for se-2-challenges repo

I think we forgot to remove it from root package.json in `base-challenge-template`  branch, just created #77 🙌

Reply:
Ok! Thank you both :) 🙌